tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp

```diff
@@ -161,7 +199,8 @@ int main(int argc, char **argv){
   ros::NodeHandle n;
 
   tf_buffer = new tf2_ros::Buffer();
-
+  tf_buffer->setUsingDedicatedThread(true);
```

Comment:
Why did you have to change this?

Reply:
When running the tests on my machine, I was getting many instances of this error:

[ERROR] [1518043130.234301067]: Do not call canTransform or lookupTransform with a timeout unless you are using another thread for populating data. Without a dedicated thread it will always timeout.  If you have a seperate thread servicing tf messages, call setUsingDedicatedThread(true) on your Buffer instance.

Adding this line eliminated the errors.

Reply:
It's not "correct" but if it suppresses the warnings/errors for the test console output that's probably better for readability.